cards/src/main/resources/cards/custom/group9/storyteller/basic/minion_dragon_librarian.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "Dragon Librarian",
+  "baseManaCost": 6,
+  "type": "MINION",
+  "heroClass": "OLIVE",
+  "baseAttack": 5,
+  "baseHp": 4,
+  "rarity": "FREE",
+  "race": "DRAGON",
+  "description": "Battlecry: Summon two 1/1 Pagelings. Draw a card.",
+  "battlecry": {
+    "targetSelection": "NONE",
+    "spell": {
+      "class": "MetaSpell",
+      "spells": [
+        {
+          "class": "SummonSpell",
+          "cards": [
+            "token_pageling",

```

Comment:
summon one `boardPositionRelative: LEFT` and another `boardPositionRelative: RIGHT`